review-notifications/src/Github.js

```diff
@@ -134,35 +180,45 @@ class Github extends Component {
   listOfPullRequests(itemNum) {
     const { classes } = this.props;
     const list = this.state.userRelatedPRList;
-
-    const header = (
-      <Grid item container class={classes.bar}>
-        <ListItem>
-          <ListItemText primary="User related pull requests" />
-        </ListItem>
-      </Grid>
-    );
     let prompt = <ListItemText dense primary="No pull requests" />;
-    if (!this.state.hasData) {
-      prompt = <ListItemText dense primary="Loading..." />;
-    } else if (this.state.user === '') {
+
+    if (this.state.user === '')
       prompt = (
         <ListItemText
           dense
           primary="Add username to display more pull requests"
         />
       );
-    } else if (this.state.token === '' || !this.state.auth) {
+    else if (this.state.token === '' || !this.state.auth)
```

Comment:
this is more of a personal opinion but if you use a lot of else/ifs, switch cases become more readable, consider it